ci-operator/config/openshift/csi-driver-shared-resource-operator/openshift-csi-driver-shared-resource-operator-release-4.10.yaml

```diff
@@ -44,10 +44,22 @@ tests:
     make verify
   container:
     from: src
-- as: e2e-aws
+- as: e2e-aws-csi-driver
   steps:
     cluster_profile: aws
-    workflow: openshift-e2e-aws
+    pre:
+    - chain: ipi-aws-pre
+    - as: deploy-operator
+      cli: latest
+      commands: make deploy
+      from: src
+      resources:
+        requests:
+          cpu: 50m
+          memory: 100Mi
```

Comment:
@gabemontero don't we need to inject the dependency for the DRIVER_IMAGE environment variable? I suspect there is a fallback in the operator which sets it to the image we were publishing to quay.io.

See https://github.com/openshift/csi-driver-shared-resource-operator/blob/master/assets/node.yaml#L45

Reply:
I believe you are right wrt this @adambkaplan, wrt validating future PRs

Quick note: I blieve it is orthogonal though to the error I'm chasing in https://github.com/openshift/release/pull/21044#issuecomment-899675528 as the that error hightlights that we are picking up a `csi-driver-shared-resource` image instead of a `csi-driver-projected-resource` image 

I'll accept your commits once my latest retest completes and I either confirm or rule out he "pipeline did not fully promote" worry

Reply:
Note that I made an edit for `NODE_DRIVER_REGISTRAR_IMAGE` - the make scripts use slightly different env vars